orbit/pkg/table/csrutil_info/csrutil_info.go

```diff
@@ -0,0 +1,67 @@
+//go:build darwin
+// +build darwin
+
+package csrutil_info
+
+import (
+	"context"
+	tbl_common "github.com/fleetdm/fleet/v4/orbit/pkg/table/common"
+	"github.com/osquery/osquery-go/plugin/table"
+	"github.com/rs/zerolog/log"
+	"os/exec"
+	"strings"
+	"syscall"
+	"time"
+)
+
+// Columns is the schema of the table.
+func Columns() []table.ColumnDefinition {
+	return []table.ColumnDefinition{
+		table.IntegerColumn("ssv_enabled"),
+	}
+}
+
+// Generate is called to return the results for the table at query time.
+// Constraints for generating can be retrieved from the queryContext.
+func Generate(ctx context.Context, queryContext table.QueryContext) ([]map[string]string, error) {
+	SSVEnabled, err := getSSVEnabled(ctx)
+	return []map[string]string{
+		{"ssv_enabled": SSVEnabled},
+	}, err
+}
+
+func getSSVEnabled(ctx context.Context) (SSVEnabled string, err error) {
+	res, err := runCommand(ctx, "/usr/bin/csrutil", "authenticated-root", "status")
+	SSVEnabled = ""
+	if err == nil {
+		SSVEnabled = "0"
+		if strings.Contains(res, "Authenticated Root status: enabled") {
+			SSVEnabled = "1"
+		}
+	}
+	return SSVEnabled, err
+}
+
+func runCommand(ctx context.Context, name string, arg ...string) (res string, err error) {
+	uid, gid, err := tbl_common.GetConsoleUidGid()
+	if err != nil {
+		log.Debug().Err(err).Msg("failed to get console user")
+		return "", err
+	}
+
+	ctx, cancel := context.WithTimeout(ctx, 5*time.Second)
+	defer cancel()
+	cmd := exec.CommandContext(ctx, name, arg...)
+
+	// Run as the current console user (otherwise we get empty results for the root user)
```

Comment:
`/usr/bin/csrutil` does not work as `root`? (Double checking if this is a copy/paste leftover from the other checks or you actually tried running csrutil as root.)

Reply:
If I understand correctly, you caught an issue for several recent tables that only need root.
I removed it from here and will remove others in a separate PR.
Let's chat to see if this removal is not needed. (otherwise keep the removal)